plugins/callback/splunk.py

```diff
@@ -154,14 +154,14 @@ def send_event(self, url, authtoken, validate_certs, include_milliseconds, batch
 
         # This wraps the json payload in and outer json event needed by Splunk
         jsondata = json.dumps(data, cls=AnsibleJSONEncoder, sort_keys=True)
-        jsondata = '{"event":' + jsondata + "}"
+        jsondata = ('{"event":' f"{jsondata}" '}')
```

Comment:
Why not keep the original version? It's easier to read than the new one IMO. Or use
```suggestion
        jsondata = '{"event":%s}' % jsondata
```
Sometimes not using f-strings is better :)

Reply:
come to think of it, in this particular case, since the content _is_ JSON, wouldn't be better/safer to:
```
        jsondata = json.dumps(dict(event=data), cls=AnsibleJSONEncoder, sort_keys=True)
```

Reply:
:+1: